### PR TITLE
Drop support for EOL Python 2.7 - 3.4

### LINKED ISSUE #383: Drop support for Python 2.7
It is time. Let's make sure there aren't any critical bugs outstanding in the master and then cut a new release for Python 3.5+ or Python 3.6+.

### PR DESCRIPTION
**Pull request by hugovk**
Fixes #383.

Python 3.4 went EOL yesterday (2019-03-16).

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,7 @@
 dist: xenial
-sudo: false
```
Comment (jaraco) [inline .travis.yml:2]: I prefer to run with `sudo: false`. Is that the default now? If so, I'd like to make that change in jaraco/skeleton.

File: keyring/backends/SecretService.py [hunk 2]
```diff
@@ -80,7 +80,7 @@ def set_password(self, service, username, password):
             "service": service,
             "username": username
         }
-        label = "Password for '%s' on '%s'" % (username, service)
+        label = "Password for '{}' on '{}'".format(username, service)
```
Comment (jaraco) [inline keyring/backends/SecretService.py:83]: I probably would have deferred these compatible changes for another time, as it's more about style than compatibility, but happy to accept with them.

**Review by jaraco — CHANGES_REQUESTED** [review]
All this needs now is a CHANGES.txt entry with a 1.x bump.

File: .travis.yml (hunk 1 above)
Comment (hugovk) [reply .travis.yml:2]: The `sudo: false` env is no longer available and the `sudo` keyword is fully deprecated:

* https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration

Would you like me to remove this commit?

**Comment by hugovk** [comment]
> All this needs now is a CHANGES.txt entry with a 1.x bump.

Updated!

File: .travis.yml (hunk 1 above)
Comment (jaraco) [reply .travis.yml:2]: Fine to leave it as submitted. Thanks for the reference.

**Review by jaraco — APPROVED** [review]

**Pull request merged**